model/Software/Classes/PackageHash.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# PackageHash
+
+## Summary
+
+A verification method for software packages represented as a single blob of bytes, such as a tar archive or single executable.
+
+## Description
+
+The PackageHash is a hash of the single blob of bytes representing a software package.
+The packageDownloadLocation should be a download locator that retrieves the package blob.  
```

Comment:
Is this referring to `downloadLocation`? Which is optional in `Package`?

Reply:
Yes - note the word "should" rather than "must" as it relates to optional.